### PR TITLE
134 fix test collection

### PR DESCRIPTION
**Pull request by merituulie**
- Removed some tests, added new ones. The test collection now focuses on creating data to the db and does not fail in case one runs the collections multiple times against database with existing data.
- The debug user to use in frontend is now user 5, username 'user', password 'password'
- We don't check for status code returned on some of the user 5 requests as they would fail when requested again with existing data on database.

From now on I suggest it to be a convention to import the collection.json and the enviroment.json to your Postman and run these tests everytime before creating the PR. If the tests fail, those need to be fixed / checked if there was a bug created.

### CODE REVIEW

File: backend/entrypoint.sh
Comment (azarSarikhani) [inline backend/entrypoint.sh:1]: amazing clean up here

**Review by azarSarikhani — APPROVED** [review]
2 thumbs up

**Review by hussojo — APPROVED** [review]

File: backend/entrypoint.sh
Comment (merituulie) [reply backend/entrypoint.sh:1]: I know, its amazing

**Review by azarSarikhani — APPROVED** [review]
Thanks for fixing the merge conflict.

**Review by dorianjagusch — APPROVED** [review]

**Review by sakarisalmi — COMMENTED** [review]
see comments

File: backend/Friends/views.py [hunk 1]
```diff
@@ -22,7 +22,6 @@ def get(self, request):
 		return Response(serializer.data, status=status.HTTP_200_OK)
 
 	@method_decorator(must_be_authenticated)
-	@method_decorator(must_be_body_user_id)
```
[the remark this answers is not in the document]
Comment (merituulie) [reply backend/Friends/views.py:25]: The friend request wont have user_id (this change was done actually a little while ago, when the friendsPage was created), so we can't validate this.

[the remark this answers is not in the document]
Comment (merituulie) [reply backend/Friends/views.py:25]: we only have friend_id in the request, as we need / can / should assume one can only add friend rows as themselves as the user.

File: backend/Friends/models.py [hunk 2]
```diff
@@ -15,9 +14,9 @@ def __str__(self):
 		return f"{self.user_id} - {self.friend_id} - {self.insertTS}"
 
 	class Meta:
-		# required because of RunTimeError that complains about lack of app_label
 		app_label = 'Friends'
 		constraints = [
-			models.UniqueConstraint(fields=['user_id', 'friend_id'], name='unique_user_friend_pair')
+			models.UniqueConstraint(fields=['user_id', 'friend_id'], name='unique_user_friend_pair'),
+			models.UniqueConstraint(fields=['friend_id', 'user_id'], name='unique_friend_user_pair'),
```
[the remark this answers is not in the document]
Comment (merituulie) [reply backend/Friends/models.py:20]: Nope, as the test collection goes through when creating friendships etc, as there are the tests for that. The constraint SHOULD actually be both ways around as we need / should only have one row for user_id - friend_id BUT ALSO only one the other way around. In any case now looking at it, might be that we dont actually need this as its just a duplicate constraint on the same fields :D Not something for this PR though, the friend keys need to be reworked anyway and I could do that next

File: backend/Friends/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sakarisalmi) [reply backend/Friends/views.py:25]: ok, that clears it up!

File: backend/Friends/models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sakarisalmi) [reply backend/Friends/models.py:20]: ok

**Review by sakarisalmi — APPROVED** [review]
questions answered!

**Pull request merged**